cedar-policy/src/api.rs

```diff
@@ -345,6 +356,10 @@ impl Entities {
     /// error if attributes have the wrong types (e.g., string instead of
     /// integer), or if required attributes are missing or superfluous
     /// attributes are provided.
+    /// # Errors
```

Comment:
We could consider enabling the [missing_error_doc lint](https://rust-lang.github.io/rust-clippy/stable/index.html#/missing_errors_doc) if we believe this sort of documentation is valuable.